theano/tensor/nnet/tests/test_conv3d2d.py

```diff
@@ -76,7 +76,10 @@ def pyconv3d(signals, filters):
 
 
 def check_diagonal_subtensor_view_traces(fn):
-    assert check_stack_trace(fn, [DiagonalSubtensor, IncDiagonalSubtensor])
+    def check_node_type(node):
+        return isinstance(node.op, (DiagonalSubtensor, IncDiagonalSubtensor))
+    assert check_stack_trace(fn, ops_to_check=check_node_type,
```

Comment:
Is there a reason to explicitly use `isinstance`?
I would find an explicit list/tuple passed as `ops_to_check` more readable, but I won't argue over it :)